src/sections/account/api-token-section/ApiTokenSection.tsx

```diff
@@ -0,0 +1,71 @@
+import { Trans, useTranslation } from 'react-i18next'
+import { Button } from '@iqss/dataverse-design-system'
+import accountStyles from '../Account.module.scss'
+import styles from './ApiTokenSection.module.scss'
+
+export const ApiTokenSection = () => {
+  const { t } = useTranslation('account', { keyPrefix: 'apiToken' })
+
+  const apiToken = '999fff-666rrr-this-is-not-a-real-token-123456'
+  const expirationDate = '2025-09-04'
+
+  const copyToClipboard = () => {
+    navigator.clipboard.writeText(apiToken).catch(
+      /* istanbul ignore next */ (error) => {
+        console.error('Failed to copy text:', error)
+      }
+    )
+  }
+
+  return (
+    <>
+      <p className={accountStyles['helper-text']}>
+        <Trans
+          t={t}
+          i18nKey="helperText"
+          components={{
+            anchor: (
+              <a
+                href="http://guides.dataverse.org/en/latest/api"
+                target="_blank"
+                rel="noreferrer"
+              />
+            )
+          }}
+        />
+      </p>
+      {apiToken ? (
+        <>
+          <p className={styles['exp-date']}>
+            {t('expirationDate')} <time dateTime={expirationDate}>{expirationDate}</time>
```

Comment:
If the expiration date is expired, we need a warning message about that, for example:

![Screenshot 2024-09-12 at 10 19 19 AM](https://github.com/user-attachments/assets/b6c486d6-b9f4-400c-8c0f-78d2262fc546)
![Screenshot 2024-09-12 at 10 19 19 AM](https://github.com/user-attachments/assets/b6c486d6-b9f4-400c-8c0f-78d2262fc546)

Reply:
Yes, that will require also a use case from this native API endpoint [Find a Token’s Expiration Date](https://guides.dataverse.org/en/latest/api/native-api.html#id162).
I prefer to add that message when we make the integration, but I will mention about this use case in [this current sprint issue](https://github.com/IQSS/dataverse-client-javascript/issues/190).